toolkit/templates/forms/_checkbox-tree-selection-buttons.html

```diff
@@ -25,12 +25,18 @@ <h3 class="categories-heading">{{ parent_category.label }}</h3>
             </legend>
 
             {% for option in parent_category.options %}
+              {% if loop.last %}
```

Comment:
You could probably pass `loop.last` directly in as a variable, or even `set is_last = loop.last` or something to save a bit of space.